test/system/030-run.bats

```diff
@@ -1233,4 +1233,18 @@ search               | $IMAGE           |
     done < <(parse_table "$tests")
 }
 
+@test "podman run umask" {
+    default_umask="0022"
+    umask="0017"
+    run_podman run -q $IMAGE grep Umask /proc/self/status
+    is "$output" "Umask:.*$default_umask" "default_umask should not be modified"
```

Comment:
Not really much point to a `default_umask` variable here, it might be more readable to just hardcode `0022` here.